src/modules/vtol_att_control/vtol_type.h

```diff
@@ -292,6 +307,18 @@ class VtolType
 
 	float update_and_get_backtransition_pitch_sp();
 
+	/**
+	 * @brief      Overrides controls for control surface or tilt mechanism in the context of preflight tests.
+	 *
+	 * @return     true while executing the test, false once done
+	 */
+	bool override_controls_for_test_mode();
+
+	SlewRate<float> _spoiler_setpoint_with_slewrate;
+	SlewRate<float> _flaps_setpoint_with_slewrate;
+
+	float _dt{0.0025f};
```

Comment:
units and description pls :)

Reply:
https://github.com/PX4/PX4-Autopilot/pull/19329/commits/3dd4403463f210743ac80a5d838a7de88f934259